index.html

```diff
@@ -1181,30 +1187,274 @@
+				Although <a data-lt="Selector">Selectors</a>, <a data-lt="Position">Positions</a>, and <a data-lt="State">States</a> 
+				provide a flexible way of identifying, e.g., a suitable <a>Segment</a> of a Resource, 
+				the fact that this is defined through an indirection using a <a>Locator</a> may be an obstacle for some applications. 
 			</p>
 
 			<p class="ednote">
 				We must add a good example. The original one referred to RDF, which is not relevant in this context.
 			</p>
 
 			<p>
-				To mitigate this issue, a mapping of <a data-lt="Selector">Selectors</a> and <a data-lt="State">States</a> on URL fragments&nbsp;[[url]] is defined below. 
-				As a result of this mapping the targeted <a>Segment</a>, or the relevant state, is expressed in a single (albeit complex) URL.
-				In that URL the <a>Selector</a>, respectively the <a>State</a>, is expressed as a single string and serves as a fragment combined with the URL of the <a>Source</a>. 
-				Note that this representation is valid only if the URL for the <a>Source</a> does not contain a fragment identifier of its own (a URL may contain at most one fragment identification).
+				To mitigate this issue, a mapping of <a data-lt="Selector">Selectors</a>, <a data-lt="Position">Positions</a>,
+				and <a data-lt="State">States</a> on URL fragments&nbsp;[[url]] is defined below. 
+				As a result of this mapping the targeted <a>Segment</a>, <a>Locus</a>, or relevant State, is expressed in a 
+				single (albeit complex) URL.
+				In that URL the <a>Selector</a>, <a>Position</a>, or <a>State</a>, is expressed as a 
+				single string and serves 
+				as a fragment which can be combined with the URL of the <a>Source</a>. 
+				Note that this representation is valid only if the URL for the <a>Source</a> does not contain a fragment 
+				identifier of its own (a URL may contain at most one fragment identification).
 			</p>
 
 			<p>
-				The syntax for mapping a <a>Selector</a>, respectively a <a>State</a>, follows the same, “functional” syntax as used, for example, by the XPointer Framework&nbsp;[[xptr-framework]]:
+				The syntax for mapping a <a>Selector</a>, <a>Position</a>, or <a>State</a> follows the same, “functional” syntax as used, 
+				for example, by the XPointer Framework&nbsp;[[xptr-framework]]:
 			</p>
 
 			<ul>
-				<li>The fragment uses the <code>selector(…)</code>, respectively the <code>state(…)</code>, functional syntax</li>
+				<li>The fragment uses the <code>selector(…)</code>, <code>position(...)</code>, or <code>state(…)</code>, functional syntax</li>
 				<li>The (comma separated) “parameters” of the functional notation are:
 					<ul>
 						<li>For the keys <code>refinedBy</code>, <code>startSelector</code>, and <code>endSelector</code> the syntax is <code>key=selector(…)</code>, respectively <code>key=state(…)</code> when appropriate, with the value following, recursively, the same syntax as the full fragment;</li>
```

Comment:
The key=position(...) should be added here as well (which means rewording the sentence a bit...)